### PR TITLE
Ana/fix testnet checkbox styles

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Sorry @iambeone , I saw your PR when I was nearly finished with this one...

Otherwise I would have committed to yours 🤷‍♀ 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
thanks ana, the checkbox still isn't aligned:

Screen Shot 2020-02-09 at 9 40 56 AM

**Comment by Bitcoinera** [comment]
> thanks ana, the checkbox still isn't aligned:
> 
> Screen Shot 2020-02-09 at 9 40 56 AM

XD no wonder I failed the CSS/Bootstrap exam

**Comment by Bitcoinera** [comment]
OK, please let me know how it looks by you after the last commit @jbibla 

Now I never see them centered. I always see some pixels of difference xD

File: src/components/common/TmSessionSignIn.vue [hunk 1]
```diff
@@ -190,19 +190,3 @@ export default {
...
-<style scoped>
-input[type="checkbox"] {
-  vertical-align: middle;
-  margin: 0.5rem 0.2rem 0.5rem 0;
-}
-.field-checkbox-testnet {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-}
```
Comment (Bitcoinera) [inline src/components/common/TmSessionSignIn.vue]: I see I had too much unnecessary CSS code here 👀

**Comment by Bitcoinera** [comment]
Let's merge this!

**Pull request merged**